packages/Tabs/src/Tabs.js

```diff
@@ -116,7 +137,7 @@ Tabs.propTypes = {
   /** Tab labels will be truncated when they run out of space instead of breaking to multiple lines (ignored when isVertical is false). */
   hasTruncation: PropTypes.bool,
 
-  /** Sets what tabindex is active (controlled component). */
+  /** Sets what tabindex is active (controlled component). Use null for no active tab. */
```

Comment:
Imo this prop should be called named `activeIndex`

Reply:
I agree. But not interested in breaking the API for this. 🤷‍♂️